Tools/gulp/tasks/compileExtensions.js

```diff
@@ -0,0 +1,52 @@
+const fs = require('fs-extra');
+const utils = require('../utils');
+
+
+
+async function compileSpellfix(dest) {
```

Comment:
Thanks for creating the compile script but how do you plan to handle Windows (which is 60% of our users)? The script will run from within PowerShell which won't have these tools.

I'm not sure the script will run successfully on CI i some dependency is missing, but I can check that when it's deployed.

The most important I guess is to make it work on Windows. If it's too complicated to build from source then we could fall back on a pre-compiled binary in this case. What do you think?